### PR TITLE
Spec for column vs image size

### PR DESCRIPTION
**Pull request by almedina-ms**
## Related Issue
Please use one of the well-known [github fixes keywords](https://help.github.com/en/articles/closing-issues-using-keywords) to reference
the issues fixed with this PR (eg Fixes #<github issue number>). If an issue doesn't yet exist please create one if reasonable to aid 
in issue tracking.

**NOTE**: For multiple issues resolved by this PR use the corresponding keywords **every time** in a comma-delimited list per the reference
page above.

## Description
Create a [descriptive title and provide all of the relevant details in the description](https://chris.beams.io/posts/git-commit/). 
This information lives with the code and will help future readers (***including yourself***) and will serve as documentation.

At the very least please describe the issue you are addressing, and what your fix entails. 

## How Verified
How you verified the fix, including one or all of the following: 
1. New unit tests that were added if any. If none were added please add a quick line explaining why not.
2. Existing relevant unit/regression tests that you ran
3. Manual scenario verification if any; ***Do include .gif's or screenshots of the testing you performed here if you think that it 
will aid in code reviews or corresponding fixes on other platforms for eg.***

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/AdaptiveCards/pull/4887)

### CODE REVIEW

**Comment by shalinijoshi19** [comment]
> ## Related Issue
> Please use one of the well-known [github fixes keywords](https://help.github.com/en/articles/closing-issues-using-keywords) to reference
> the issues fixed with this PR (eg Fixes #). If an issue doesn't yet exist please create one if reasonable to aid
> in issue tracking.
> 
> **NOTE**: For multiple issues resolved by this PR use the corresponding keywords **every time** in a comma-delimited list per the reference
> page above.
> 
> ## Description
> Create a [descriptive title and provide all of the relevant details in the description](https://chris.beams.io/posts/git-commit/).
> This information lives with the code and will help future readers (_**including yourself**_) and will serve as documentation.
> 
> At the very least please describe the issue you are addressing, and what your fix entails.
> 
> ## How Verified
> How you verified the fix, including one or all of the following:
> 
> 1. New unit tests that were added if any. If none were added please add a quick line explaining why not.
> 2. Existing relevant unit/regression tests that you ran
> 3. Manual scenario verification if any; _**Do include .gif's or screenshots of the testing you performed here if you think that it
>    will aid in code reviews or corresponding fixes on other platforms for eg.**_
> 
> ###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/AdaptiveCards/pull/4887)

@almedina-ms can you update the description on this PR and remove the template placeholder text? Thanks

File: specs/DesignDiscussions/ImagevColumn.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    - [Current Problem](#current-problem)
+    - [What's happening behind the scenes?](#whats-happening-behind-the-scenes)
+    - [Proposed Solutions](#proposed-solutions)
+        - [Honoring Image Size](#honoring-image-size)
+
+## Overview
+
+### Current problem
+
+As part of the solution to the Column width behaviour it was thought that images could be used as a reference to provide the minimum size needed to render an `auto` width column. This asseveration is not correct, as some of our scenario cards don't follow that principle, take for example the [FlightDetails card](), in this card we have an image whose `width` is `1000px`, if we were to honour the measurements given by the card author we would get a card where this long image occupies most of the width of the column set as can be seen in the views generated by the iOS and the Android renderers.
```
Comment (shalinijoshi19) [inline specs/DesignDiscussions/ImagevColumn.md:14]: Thanks for trying to add context here. But can you also link to the "Column width behavior" issue that you are referring to here for clarification? Thanks

Comment (shalinijoshi19) [inline specs/DesignDiscussions/ImagevColumn.md:14]: :) I had to look up the dictionary for that one :) - Could we use 'assumption' here?

Comment (shalinijoshi19) [inline specs/DesignDiscussions/ImagevColumn.md:14]:
```suggestion
As part of the solution to the Column width behavior it was thought that images could be used as a reference to provide the minimum size needed to render an `auto` width column. This assumption is not correct as some of our scenario cards don't follow that principle. Take for example the [FlightDetails card](), where we have an image whose `width` is `1000px`; If we were to honor the measurements given by the card author we would get a card where this long image occupies most of the width of the column set as can be seen in the views generated by the iOS and the Android renderers.
```

Comment (shalinijoshi19) [reply specs/DesignDiscussions/ImagevColumn.md:14]: Is it https://github.com/microsoft/AdaptiveCards/pull/4722 ?

File: specs/DesignDiscussions/ImagevColumn.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+| iOS | <img src="assets/ImageColumn/iOSFlightDetails.png" width="400px"> |
+| Android | <img src="assets/ImageColumn/androidFlightDetails.png" width="400px"> |
+
+### What's happening behind the scenes?
+
+As we have seen, Javascript is generating the better looking card, but how does it do it? As we can see in the columnset, there are other columns that have images on them so why don't they shrink as well? The difference between columns 1, 3, 4 and column 2 is that the second column has `stretch` width, so after changing that width to `auto` we got a similar behaviour to the iOS and Android platforms: 
+
+| Rendered view | Schema
+| --- | --- |
+| <img src="assets/ImageColumn/BadFlightDetails.png" width="800px"> | <img src="assets/ImageColumn/BadRenderSchema.png" width="800px"> |
```
Comment (shalinijoshi19) [inline specs/DesignDiscussions/ImagevColumn.md:28]: This results in a 404.. Does this need to include the relative path here vis "..\assets"?

**Review by jwoo-msft — APPROVED** [review]
Hey,

Excellent work! This spec will really help. In case we forget to check-this in. I'm adding the approval, so this PR will get check-in.

**Pull request merged**